apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SimpleSubscriber.cs

```diff
@@ -357,15 +357,15 @@ internal SimpleSubscriberImpl(SubscriptionName subscriptionName, IEnumerable<Sub
             _scheduler = settings.Scheduler ?? SystemScheduler.Instance;
             _taskHelper = GaxPreconditions.CheckNotNull(taskHelper, nameof(taskHelper));
             _flowControlSettings = settings.FlowControlSettings ?? DefaultFlowControlSettings;
-            _maxAckExtendCount = (int)Math.Min(_flowControlSettings.MaxOutstandingElementCount ?? long.MaxValue, 10_000);
+            _maxAckExtendQueue = (int)Math.Min(_flowControlSettings.MaxOutstandingElementCount ?? long.MaxValue, 20_000);
```

Comment:
Why the rename? It does sound like a count.

Reply:
It used to be the count of ack/extend messages pushed within each batch to the server.
It's now the max size of the queue on the client. It should really be called `maxAckExtendQueueSize`. I'll change it then push on green.